scripts/setup-mapbox-sdk-walkthrough.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Mapbox SDK Credential Setup Script
+# ==================================
+#
+# Purpose:
+# --------
+# This script assists users in setting up the necessary credentials to utilize
+# Mapbox's closed-source SDKs for iOS and Android. It provides step-by-step
+# guidance for obtaining a secret token from Mapbox and subsequently invokes
+# the "setup-mapbox-sdk.sh" script to configure the development environment.
+#
+# Background:
+# -----------
+# To use the Mapbox SDKs for iOS and Android development, a secret token
+# must be obtained from Mapbox's account page. This token is essential for
+# authenticating downloads of the closed-source SDKs during the build process.
+#
+# Usage:
+# ------
+# To configure Mapbox, invoke this script by running the following command from the project's root directory:
+# npm run configure-mapbox
+
+# Use functions and varaibles from the utils script
+source scripts/shellUtils.sh
+
+# Intro message
+title "This script helps you set up the credential needed to use Mapbox's closed-sourced SDKs for iOS and Android."
+echo -e "\n"
+
+echo -e "1. Visit: https://account.mapbox.com/access-tokens/\n"
```

Comment:
You can use `info` function from `shellUtils` instead of `echo -e` if you want.

Reply:
`info` function makes text blue so I decided to keep `echo -e` for these texts 👍